_i18n/nb-no/resources/user-guides/multisig-messaging-system.md

```diff
@@ -0,0 +1,864 @@
+
+Hvor stor denne faren ved å etterlikne er, avhenger av hvor sikker den opprinnelige utvekslingen av nøkkelsett er ved begynnelsen av hele prosessen når man konfigurerer lommebøkene og endelig «går multisig»: Hvis du kan forsikre om at kun de riktige personene får de riktige nøkkelsettene og ingen kan utgi seg for å være en annen, er alt i orden. Hvis ikke, kan du tape.
+
+Hvis du bruker alle kapabilitetene til MMS-en, bruker du den ikke bare til å foreta transaksjoner med, men allerede før for å utveksle nøkkelsett mellom alle signerere. Spesielt for høyere former for multisig som 2/4 med flere runder med nøkkelutveksling, er dette svært hjelpsomt og innehar en mindre fare for å begå feil enn noen manuelle prosesser. Så oppgaven med å forebygge etterlikning går over fra å sikre utvekslingen av nøkler til å sette opp en signerer-adresse i MMS-en på en sikker måte: Hvis Bob noen måte kan lure Alice til å godta én av **hans** Monero- og Bitmessage-adresser i stedet for Trent sine, har han vunnet.
+
+De tre måtene å sette opp signereradresser som MMS-en støtter, manuelt konfigurering av signerere, auto-config og «semiautomatisk» sending av fullført signererinformasjon, har alle forskjellige risikoer knyttet til dem hva gjelder etterlikning. Sjekk de respektive kapitlene *Manuell konfigurering av signerere*, *Auto-Config* og *Sending av signererinformasjon* for mer informasjon om dette.
+
+Auto-config er utvilsomt det enkleste å sikre: Du trenger bare en liten del med informasjon og en auto-config-token på 11 tegn for å sende sikkert til hver signerer, og hvis du kan gjøre det, har du allerede vunnet. («Config-lederen» antar selvsagt å være pålitelig.)
+
+Hvis alt dette høres for komplisert og derfor upålitelig ut for deg, har du muligheten til å konfigurere lommebøker og etablere multisig-adressen mens du utelater MMS-en helt og kun bruker den senere for å komfortabelt sende delvis signerte transaksjoner rundt og avløse deg fra de langtekkelige, manuelle synkroniseringene av lommebøkene etter hver transaksjon.
+
+### Angrepskontrollerte data
+
+Det er to situasjoner der lommeboken din som bruker MMS mottar data fra en annen signerer, der den andre signereren, hvis vedkommende handler med onde hensikter, kan prøve å bedra deg eller lure deg til å gjøre noe skadelig:
+
+Notater som sendes med `mms note`-kommandoen kan brukes for «sosialteknikk». En angriper kunne f.eks. ha prøvd å formulere et notat som ser ut som en feilmelding i et forsøk på å svindle. De tekniske mulighetene her er imidlertid ganske begrensende: Notater begrenser seg strengt til tekst, og når de vises filterer MMS-en ut tegn med ASCII-koder på mindre enn 32 og de to tegnene < og > som kan brukes til å bygge HTML eller XML som på en eller annen måte kan tolkes (veldig usannsynlig i CLI-lommeboken, men noe mer sannsynlig i GUI-baserte lommebøker). Notatene har også en grense på maksimal lengde.

```

Comment:
Maybe change to: "Notater begrenser seg strengt til tekst, og når de vises, filtrerer MMS-en ut tegn som inneholder ASCII-koder som har en lengde på under …"

Reply:
fixed